server/etcdserver/api/etcdhttp/health.go

```diff
@@ -266,7 +295,9 @@ func (reg *CheckRegistry) installRootHttpEndpoint(lg *zap.Logger, mux *http.Serv
 		excluded := getQuerySet(r, "exclude")
 
 		filteredCheckNames := filterCheckList(lg, listToStringSet(checks), excluded)
-		return reg.runHealthChecks(r.Context(), filteredCheckNames...)
+		h := reg.runHealthChecks(r.Context(), filteredCheckNames...)
+		recordMetrics(reg.path, "/", h.Status)
```

Comment:
change the "/" to "default" or "root"? 

Also a related suggestion: can we define a list of enum for all checks? This can be discussed and addressed in a separate session/PR.

```suggestion
		recordMetrics(reg.path, "default", h.Status)
```

Reply:
changed to root.